### ISSUE TITLE
Break times

### ISSUE THREAD

**Issue by ragamouf** [au-research/github-training#3]
If this stays at 4 hours, I think we should adjust timings to be 1 hour total break time, eg 15 mins plus 45 mins.
Considering how we might typically run this for 4 timezones

| 9-1 | WA |
| 10:30 - 2:30 | SA, NT |
| 11-2 | Eastern states |
| 1-5 | NZ |

**Comment by orchid00** [comment]
@mpfl this might be a comment imported from the other repo, please check and delete

**Comment by mpfl** [comment]
@orchid00 It's not. The import/clone process shouldn't copy Issues across.

**Comment by mpfl** [comment]
@ragamouf I don't think we should prescribe the timing of breaks in the lesson material - let the instructor do that when they instantiate the materials into a workshop. Instead, focus on making the materials require only 3:20 or 3:30 to cover.

**Comment by ragamouf** [comment]
@mpfl I see your point, but I think it is important to include suggested breaks, for example throughout the carpentries lessons breaks are a regular feature. The advantage of including breaks mean you can identify an overall duration, while also specifying teaching time. This is just as important for workshop organisers as learners in managing expectations.

The problem as it stands is that 4 hours is spurious to what we delivered in May, so I want to clarify is 4 hours still a realistic goal for teaching time or overall duration? (Thinking about the edits and additional content we wanted to cover).

**Comment by mpfl** [comment]
Sorry, @ragamouf, but could you please point me in the direction of Carpentries lessons that suggest breaks..? I've just checked half a dozen lessons and not a single one includes suggested breaks. They only outline the teaching time required (as inaccurate as that is, sometimes).

**Comment by ragamouf** [comment]
@mpfl the instructor training curriculum: https://carpentries.github.io/instructor-training/

I believe I've outlined good reasons for including suggested break times. 

Shall we agree to discuss this issue in our coworking session later in June? For me, solving this issue relates directly to how we shape the content for this workshop.
